pkg/operators/cli/clioperator.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 The Inspektor Gadget authors
+// Copyright 2024-25 The Inspektor Gadget authors
```

Comment:
I think we use the full year everywhere else
```suggestion
// Copyright 2024-2025 The Inspektor Gadget authors
```